ludwig/experiment.py

```diff
@@ -335,6 +335,12 @@ def kfold_cross_validate(
 
     logger.info('starting {:d}-fold cross validation'.format(k_fold))
 
+    # extract out model definition for use
+    if model_definition_file is not None:
```

Comment:
maybe if you use `experiment` instead of `train` and then `predict` you don't need to manually load the model definition.

Reply:
converted to use functions out of `experiment.py`